cartographer-config.cmake.in

```diff
@@ -32,4 +32,10 @@ else ()
    find_package(Ceres HINTS ${CERES_DIR_HINTS})
 endif()
 
+set(BUILD_GRPC @BUILD_GRPC@)
```

Comment:
My understanding is that this variable is read when cartographer_ros does find_package(cartographer).
So it may potentially conflict with cartographer_ros's `option(build_grpc)`.
How about a rename to `CARTOGRAPHER_HAS_GRPC`.
Then, cartographer_ros can check if cartographer is compatible.